Lib/test/datetimetester.py

```diff
@@ -2501,6 +2501,39 @@ def test_strptime(self):
                 self.assertEqual(expected, got)
 
         strptime = self.theclass.strptime
+
+        # bpo-34903: Check that single digit dates and times are allowed.
```

Comment:
If you have a comment to delim start/end of your block, it means that you have to add a new method, like:

```
def test_strptime_single_digit():
  # bpo-34903: Check that single digit dates and times are allowed.
  ...
```

Reply:
I'm happy to make this change but note that the code immediately preceding mine has a similar comment for bpo-34482. Could you (@vstinner) and @pganssle please confirm?

Reply:
In that case, move also the previous lines into a subtest (new method).

Reply:
@mikegleen It's fairly common for there to be stylistic and organizational inconsistencies in a file, because the codebase is large, old and has many contributors with different priorities (and different reviewers with different standards).

I tend to agree with Victor here. The main reason I didn't ask for this to be in a new test to start with was because, as you said, the current test suite tends towards these single monolithic tests and I prefer smaller tests and I wasn't sure if monolithic tests was a general policy or not. Given that Victor has much more experience than me with the project's conventions, I'm happy that he asked for this, because it will make me feel more confident in making smaller tests in the future. :smile:

Also, as an aside, I'd like to thank you for your ongoing efforts in this PR, I think we're in the home stretch :wink: